components/modals/FilterModal.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <b-modal :ref="modalRef" title="Filter runs" class="text-center" @hidden="destroyModal" hide-footer centered
+           size="lg">
```

Comment:
Time selectors are a bit too small right now
```suggestion
           size="xl">
```